### DISCUSSION TITLE
Dialog and Notification Icons errors on production

### DISCUSSION THREAD

**Discussion by insulae** [wireui/wireui#703] [BUG]
I have a weird error when I put my app into production.
Everything works perfect but the icons in dialog and notification are not shown in the rest of the components yes

![image](https://github.com/wireui/wireui/assets/18574307/5061fa94-2763-4fcb-a8e8-abab40683286)

In the console I have the following error:

![image](https://github.com/wireui/wireui/assets/18574307/820a5ef3-4fa4-453b-985e-45b90b6b9535)

`fetch("/wireui/icons/".concat(null !== (e = t.style) && void 0 !== e ? e : "outline", "/").concat(t.name), {`

thanks for the help!

**Comment by insulae** [comment]
searching a bit I found this answer for a similar problem, in my case it is like this, I have my laravel/livewire application in a subdirectory (dominio.com/application)

Is your laravel app url with the subfolder path?

_Originally posted by @PH7-Jack in https://github.com/wireui/wireui/discussions/241#discussioncomment-3282379_

**Comment by insulae** [comment]
ok I have a solution for those who are desperate. but I think it is a solution that should be considered a bug. If you are working your application in a subfolder you must edit the file:

`vendor/wireui/wireui/dist/wireui.js `

and search for /wireui/ will find 3 matches: 
/wireui/icons x2 
/wireui/button 
you must add the subfolder, eg: 

`/subfolder/wireui/icons`

after this run:

`npm run build `

again and it will work correctly.

It would be nice if at least in the config/wireui.php we can add the subfolder like livewire config file have for these cases:
'asset_url' => '/subfolder,

![image](https://github.com/wireui/wireui/assets/18574307/4d4a527b-b227-4a89-9643-5670e2ce1c98)

**Reply by blockhunts** [reply]
nice, thank you
